src/sst/elements/memHierarchy/memoryController.cc

```diff
@@ -589,6 +594,12 @@ void MemController::writeData(MemEvent* event) {
         }
 
         backing_->set(addr, event->getSize(), event->getPayload());
+        if (is_debug_event(event)) {
+            Debug(_L10_, "\tContents ");
+            for( auto i : event->getPayload() )
+                std::cerr << uint32_t(i) << " ";
+            std::cerr << "\n";
+        }
```

Comment:
The call to printDataValue() above (line 593) prints the payload if debug level is 11. Is that not enough? Same with below (line 611 prints)

Reply:
Let me go look. I wrote this ~2y ago and I didn't see this functionality then.

Reply:
Removed